py/server/deephaven/table_factory.py

```diff
@@ -3,13 +3,14 @@
 #
 
 """ This module provides various ways to make a Deephaven table. """
-from typing import List, Dict, Any, Union
+from __future__ import annotations
```

Comment:
What's this import used for?

Reply:
it was needed for an earlier iteration, no longer needed but auto-import-org didn't remove it for some reason.